docs/content/getting-started.md

```diff
@@ -212,22 +195,11 @@ oc scale nodepool/$NODEPOOL_NAME \
   --replicas=$NODEPOOL_REPLICAS
 ```
 
-## Delete a HostedCluster
-
-To delete a HostedCluster:
+## Delete a Hosted Cluster
+To delete a Hosted Cluster:
 
 ```shell
 hypershift destroy cluster aws \
   --name $CLUSTER_NAME \
   --aws-creds $AWS_CREDS
 ```
-
```

Comment:
I don't think we need this section anymore since this is set to true by default, but I can put it back if we want.